CHANGES

```diff
@@ -1,3 +1,18 @@
+shutter (0.95)
+
+* almost fully dropped the dependency on the outdated Perl Gnome2 library
+* updated translations
+
+* removed dependencies:
+
+    --  Arch: gnome-perl, gnome-vfs-perl
+    --  Debian: libgnome2-perl, libgnome2-vfs-perl
+
+* new dependencies:
+
+    --  Arch: perl-number-bytes-human, perl-glib-object-introspection
```

Comment:
I don't know arch, but is perl prepended or appended?

Reply:
That's fine, I actually maintain the Arch package, here is the AUR page with the full dependency list: https://aur.archlinux.org/packages/shutter-git/ I will have a look at the README and website, thanks for pointing that out!